vm/vm.go

```diff
@@ -291,7 +296,15 @@ func (vm *VM) Run(program *Program, env interface{}) (out interface{}, err error
 				in[i] = vm.pop()
 			}
 			fn := FetchFn(env, call.Name).Interface()
-			vm.push(fn.(func(...interface{}) interface{})(in...))
+			if typed, ok := fn.(func(...interface{}) interface{}); ok {
+				vm.push(typed(in...))
+			} else if typed, ok := fn.(func(...interface{}) (interface{}, error)); ok {
```

Comment:
CallFast is a special case. We don't want to change the signature of call fast methods. The idea for call fast funcs is what they are created manually for

Reply:
We don't change it here, but rather allowing to add a new one, that also returns an error.

So both variants will work.